README.md

```diff
@@ -38,9 +38,18 @@ When running `localizer expose <serviceName>` your local machine will look for a
 Kubernetes cluster, and if it exists it will create a container that will proxy traffic sent to it to your local machine
 allowing remote resources to access your local machine as if they were also running locally.
 
+## Install `localizer`
+
+You can install the (OSX/LINUX) binary directly into /usr/local/bin:
+```
+$ curl -Ls https://github.com/jaredallard/localizer/releases/download/v1.1.1/localizer_1.1.1_$(uname)_amd64.tar.gz | tar  -xzC /usr/local/bin localizer
```

Comment:
I'll need to make a mental note to update this on each release, but good idea!

Reply:
If you change the naming of release artifacts to exclude version number from the tar.gz. than it can point to `latest`.
The download url will contain the version number anyways. So the 2 fix url could be:
- https://github.com/jaredallard/localizer/releases/download/v1.1.1/localizer_darwin_amd64.tar.gz
- https://github.com/jaredallard/localizer/releases/download/v1.1.1/localizer_linux_amd64.tar.gz